### PR TITLE
Add idlharness test for CSS Device Adaptation

### PR DESCRIPTION
**Pull request by lukebjerring**
Uses the IDL file from https://github.com/web-platform-tests/wpt/pull/12911.

### CODE REVIEW

**Comment by w3c-bots** [comment]
# Build PASSED

Started: 2018-03-02 21:27:13
Finished: 2018-03-02 21:37:41


<h2>Failing Jobs</h2>
<ul>

<li>chrome:dev</li>

<li>safari:11.0</li>

<li>MicrosoftEdge:16.16299</li>

</ul>




View more information about this build on:

- [WPT PR Status](http://pulls.web-platform-tests.org/build/25168)
- [TravisCI](https://travis-ci.org/w3c/web-platform-tests/builds/348381577)

**Comment by lukebjerring** [comment]
@ChumpChief - I don't seem to be able to add you as a reviewer. @foolip do you know why that is?

**Comment by foolip** [comment]
That's because @ChumpChief isn't a member of the web-platform-tests org. I've sent an invite which can be accepted at https://github.com/web-platform-tests.

File: css/css-device-adapt/META.yml [hunk 1]
```diff
@@ -0,0 +1,4 @@
+spec: https://drafts.csswg.org/css-device-adapt/
+suggested_reviewers:
+  - ChumpChief
+  - frivoal
```
Comment (foolip) [inline css/css-device-adapt/META.yml]: @frivoal I see you are listed as editor of CSS Device Adaptation, would you like to get reviews for this directory?

File: css/css-device-adapt/META.yml [hunk 2]
```diff
@@ -0,0 +1,2 @@
+suggested_reviewers:
+  - ChumpChief
```
Comment (foolip) [inline css/css-device-adapt/META.yml]: I've poked @ChumpChief but got no reaction. Luke, let's just put ourselves here. If implementer interest for this spec materializes (it was implemented in Presto, FWIW) we can hand it off.

**Review by foolip — APPROVED** [review]

**Comment by foolip** [comment]
@ChumpChief has confirmed over email he's OK with this, so I'll change META.yml back.

**Pull request merged**